app.rb

```diff
@@ -0,0 +1,202 @@
+    puts('7 - Exit')
+    puts('')
+  end
+
+  def display
+    options
+  end
+
+  def run_choise(input)
+    case input
+    when '1'
+      list_books
+    when '2'
+      list_people
+    when '3'
+      create_people
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_of_rentals
+    else
+      exit
+    end
+  end
+
+  def run
+    options
+    puts 'Enter number'
+    input = gets.chomp
+    run_choise(input)
+  end
+
+  # list book
+  def list_books
+    puts 'The Library is empty ,There is no books added yet!' if @book.length.zero?
+
+    @book.each do |book|
+      puts "#{@book.index(book)}) Title: \"#{book.title}\", Author: #{book.author}"
+    end
+
+    puts ''
+    run
+  end
+
+  # list people
+  def list_people
+    puts 'There is person in the list!' if @book.length.zero?
+
+    @people.each do |person|
+      puts "Name: #{person.name}  Age: #{person.age}"
+    end
+
+    puts ''
+    run
+  end
+
+  #   Create person
+  def create_people
+    puts 'Do you want to create a student (1) or a teacher (2) ? [Input the number]:'
+    input = gets.chomp.to_i
+    case input
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else
+      puts 'Please Enter 1/2 please'
+      create_people
+    end
+  end
+
+  # create student
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+
+    parent_permission = ''
+
+    case gets.chomp.downcase
+    when 'y'
+      parent_permission = true
+
+    when 'n'
+      parent_permission = false
+    else
+      puts('please enter [Y/N]: ')
+    end
+
+    new_student = Student.new(age, name, parent_permission)
+    @people << new_student
+    puts @people
+    run
+
+    puts('Student person created successfully!')
+  end
+
+  # create teacher
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Specialization: '
+    specialization = gets.chomp
+
+    new_teacher = Teacher.new(specialization, age, name)
+    @people << new_teacher
+    puts('Teacher person created successfully!')
+    run
+  end
+
+  # create book
+  def create_book
+    puts 'Title:'
+    title = gets.chomp
+
+    puts 'Author:'
+    author = gets.chomp
+
+    new_book = Books.new(title, author)
+    @book << new_book
+
+    puts('Book created successfully')
+    run
+  end
+
+  # create rental
+  def create_rental
+    puts('Select a book from the following list by number')
+    @book.each do |book|
+      puts("#{@book.index(book)} Title: \"#{book.title}\", Age: #{book.author}")
+    end
+
+    book_id = gets.chomp.to_i
+    selected_book = @book[book_id]
+
+    puts('Select a person from the following list by number')
+    @people.each do |person|
+      puts("#{@people.index(person)}) Name: \"#{person.name}\", Age: #{person.age}")
+    end
```

Comment:
- [ ] When I try to select a person for which I want to create a rental, the app crashes. Kindly work on that so one can be able to create a rental

**Below is a screenshot of the error when failing to create a rental**
![image](https://user-images.githubusercontent.com/50941074/186912635-d5c146ce-fe4e-4c3d-bf3c-f692a7195ed7.png)